packages/ado-extension/package.json

```
@@ -14,7 +14,7 @@
         "start": "node scripts/run-locally.js",
         "test": "jest --rootDir src",
         "watch:test": "jest --watch",
-        "test:e2e": "jest --rootDir e2e --coverage false",
+        "test:e2e": "concurrently \"npm --prefix ../test-website-hashroute start\" \"jest --rootDir e2e --coverage false\" --kill-others --success first",
```

We can avoid pulling in an additional library by using `npm-run-all` and splitting this into two commands:
```suggestion
        "test:e2e": "npm-run-all --parallel test:e2e:serve-hashroute test:e2e:run-tests",
        "test:e2e:run-tests": "jest --rootDir e2e --coverage false",
        "test:e2e:serve-hashroute": "yarn workspace test-website-hashroute start",
```

---

Removed concurrently and used npm-run-all